packages/guides/src/Renderer/LatexRenderer.php

```diff
@@ -28,7 +31,7 @@ public function supports(string $outputFormat): bool
 
     public function render(RenderCommand $renderCommand): void
     {
-        $renderDocumentHandler = new RenderDocumentHandler($this->renderer);
+        $renderDocumentHandler = new RenderDocumentHandler($this->renderer, $this->eventDispatcher);
```

Comment:
This should also be removed as you did in the HtmlRenderer by using the commandbus.

Reply:
@jaapio I don't really get yet what the command bus is doing and how to use it.

Reply:
The commandbus is routing a command to a handler. 
Some background info:
https://barryvanveen.nl/articles/49-what-is-a-command-bus-and-why-should-you-use-it